package.json

```diff
@@ -29,15 +29,20 @@
   "homepage": "https://github.com/hoodiehq/hoodie-server-store#readme",
   "devDependencies": {
     "coveralls": "^2.11.6",
+    "memdown": "^1.1.2",
     "nock": "^7.0.1",
     "nyc": "^5.3.0",
+    "request": "^2.69.0",
     "semantic-release": "^6.0.3",
     "standard": "^6.0.4",
     "tap": "^5.0.1"
   },
   "dependencies": {
+    "boom": "^3.1.2",
+    "express": "^4.13.4",
+    "express-pouchdb": "^1.0.2",
     "h2o2": "^5.0.0",
     "hapi": "^13.0.0",
-    "joi": "^8.0.1"
+    "pouchdb": "^5.2.1"
```

Comment:
I wonder if we could make `"pouchdb"` not a dependency, like we do in https://github.com/hoodiehq/hoodie-server-account? In `hoodie-server-account` we accept a `db` instance which can either use CouchDB or levelDB (or whatever) as its backend: https://github.com/hoodiehq/hoodie-server-account/tree/master/plugin#optionsusersdb

in `hoodie-server-store` we could accept options.PouchDB which would already be `PouchDB.defaults(options.pouchdb)` instead of doing that within `hoodie-server-store`?